src/CocktailCard/CocktailCard.tsx

```diff
@@ -10,11 +11,11 @@ export interface CocktailsCardProps {
 const CocktailCard: React.SFC<CocktailsCardProps> = (props) => {
   return (
     <section className="cocktail-card">
-      <div className='card-title'>
-        <h3>{props.strDrink}</h3>
-        <button className="cc-button">Make Me!</button>
-      </div>
-      <img className="CC-img" src={props.strDrinkThumb} alt={props.strDrink} />
+      <h3 className='card-title'>{props.strDrink}</h3>
+			<section className='img-container'>
+				<Link to={`/${props.idDrink}/details`}><button className="cc-button">Make Me!</button></Link>
```

Comment:
I have this in my branch that I was working on today!  I will stash it - I should have pushed it up this AM when I had it done.

Reply:
Oh sorry! I just wanted to click through the whole user story while testing out the styling @Melizzo

Reply:
Totally makes sense.  I should just push up as often as I can, even if it's 1 line of code.